lib/template-mixins.js

```diff
@@ -254,6 +254,7 @@ module.exports = function (options, deprecated) {
                 legend: t(legendValue),
                 legendClassName: legendClassName,
                 role: opts.type === 'radio' ? 'radiogroup' : 'group',
+                ariaRequired: opts.type === 'radio' ? 'true' : '',
```

Comment:
We probably want to also check that the radio group is actually required before we add the attribute, but I think that's a separate issue to this.